Zscaler enhancement - added get_ids_and_names_only argument (#12822)
demisto/content: Packs/Zscaler/Integrations/Zscaler/README.md

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |

> | displayURL | Whether to display the URLs of each category in the War Room. Default is 'false'. Anyway URLS will be returned to the Context Data. | Optional | 

Reviewer: ```suggestion
| displayURL | Whether to display the URLs of each category in the War Room. Default is 'false'. URLs will always be returned to the Context Data. | Optional | 
```

Author: Done

| custom_categories_only | Whether to retrieve only custom categories to the War Room. Default is 'false'. | Optional |

> | get_ids_and_names_only | Whether to retrieve only a list of URL category IDs and names. URLs will not be returned at all (even if displayURL set to true). Note - he API does not support the combination of custom_only and get_ids_and_names_only. | Optional |

Reviewer 2: ```suggestion
| get_ids_and_names_only | Whether to retrieve only a list containing URL category IDs and names. Even if *displayURL* is set to true, URLs will not be returned. Please note - the API does not support the combination of custom_only and get_ids_and_names_only. | Optional |
```

@abaumgarten WDYT?

Author: Perfect
